app/test_main.py

```diff
@@ -1,3 +1,51 @@
+from typing import Any
+
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_result",
+    [
+        ("str", [], TypeError)
```

Comment:
Add more test cases with different data types to utilize the functionality of `parametrize`.